abs.yaml

```diff
@@ -1,2 +1,6 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/postgresql-feedstock/pr10/29d70f6
+  - https://staging.continuum.io/prefect/fs/gdal-feedstock/pr28/929dd3a
+
 upload_channels:
   - sfe1ed40
```

Comment:
I'm not seeing pdal existing on pkgs/main when doing a `conda search` for it. I assume, since this was the state the project was left in last, that this is built for snowflake. Does that seem right?

Reply:
@bwingconda correct, it goes to Snowflake because it was previously deployed on Snowflake already with `v2.5.3` ( https://anaconda.org/sfe1ed40/pdal ) CC @Arishamays1

Reply:
I've updated the ticket's description. The destination channel was already updated with SF.